src/components/shell/flyouts/manageDeviceGroups/views/deviceGroupForm.js

```diff
@@ -122,15 +123,22 @@ class DeviceGroupForm extends LinkedComponent {
     );
   };
 
-  addCondition = () => this.conditionsLink.set([
+  addCondition = () => {
+    this.props.logEvent(toDiagnosticsModel('DeviceGroup_AddCondition', {}));
+    this.conditionsLink.set([
     ...this.conditionsLink.value,
     newCondition()
-  ]);
+    ]);
+  }
 
   deleteCondition = (index) =>
-    () => this.conditionsLink.set(this.conditionsLink.value.filter((_, idx) => index !== idx));
+    () => {
+      this.props.logEvent(toDiagnosticsModel('DeviceGroup_RemoveCondition', {}));
+      this.conditionsLink.set(this.conditionsLink.value.filter((_, idx) => index !== idx));
```

Comment:
Does the code rely on this returning anything? The old code did return something; the new doesn't.